snekbox/api/resources/eval.py

```diff
@@ -61,7 +59,14 @@ def on_post(self, req: falcon.Request, resp: falcon.Response) -> None:
 
         >>> {
         ...     "stdout": "10000 loops, best of 5: 23.8 usec per loop\n",
-        ...     "returncode": 0
+        ...     "returncode": 0,
+        ...     "attachments": [
+        ...         {
+        ...             "name": "output.png",
+        ...             "mime": "image/png",
```

Comment:
Same for size and compression type. If we're only using one compression type then there's no need to include it in the response (and I think supporting multiple compression types is overkill unless there's a real need for it). And with the size, is it really needed?

Reply:
Since we're planning on having a higher file size limit on the snekbox end and doing so on the bot end, it might be a waste of resources to uncompress a very large and well compressed content just to discover it's over our discord-side limit. Some 48MB of repetitive strings could compress to ~4KB, for example.

Reply:
Removed mime and compression in [ba6d0a8](https://github.com/python-discord/snekbox/pull/159/commits/ba6d0a8a10af687393134fc1e9662100ce67df52)